app/src/main/java/com/sun/efood/ui/detail/MealDetailActivity.kt

```diff
@@ -15,16 +14,93 @@ import com.sun.efood.databinding.ActivityMealDetailBinding
 import com.sun.efood.utils.Constants
 import com.sun.efood.utils.showToast
 
-class MealDetailActivity : YouTubeBaseActivity() {
+class MealDetailActivity : YouTubeBaseActivity(), MealDetailContact.View {
 
     private lateinit var binding: ActivityMealDetailBinding
+    private var presenter: MealDetailPresenter? = null
+    private var meal: Meal? = null
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityMealDetailBinding.inflate(layoutInflater)
         setContentView(binding.root)
+        initData()
+        initEvent()
     }
 
+    override fun showMealDetail(mealDetails: List<MealDetail>) {
+        mealDetails.first {
+            binding.apply {
```

Comment:
sao lại để trong first block thế, 
```suggestion
        val meal = mealDetails.firstOrNull() ?: return 
            binding.apply {
```